### PR TITLE
add .gitignore

### PR DESCRIPTION
**Pull request by werdoJC**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+__pycache__
+*.pyc
+*.pyo
```
Comment (LordSalfo) [inline .gitignore]: Please add an empty space on every last line on every code

**Comment by LordSalfo** [comment]
Please create another PR for adding Images. 
PRs:
1 - Add .gitignore
2 - Add images

**Comment by werdoJC** [comment]
@LordSalfo code updated

**Comment by LordSalfo** [comment]
LGTM :+1:

**Pull request merged**